tests/Unit/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/VerifyEinsteinSolution.cpp

```diff
@@ -0,0 +1,261 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/VerifyEinsteinSolution.hpp"
+
+#include "DataStructures/DataBoxTag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Index.hpp"
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/CoordinateMaps/AffineMap.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.hpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Equations.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/GrTags.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Christoffel.hpp"
+#include "PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp"
+#include "PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp"
+#include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
+#include "Utilities/TypeTraits.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+// Shorter names for tags.
+using SpacetimeMetric = gr::Tags::SpacetimeMetric<3, Frame::Inertial>;
+using Pi = ::GeneralizedHarmonic::Pi<3, Frame::Inertial>;
+using Phi = ::GeneralizedHarmonic::Phi<3, Frame::Inertial>;
+using GaugeH = ::GeneralizedHarmonic::GaugeH<3, Frame::Inertial>;
+
+using VariablesTags = typelist<SpacetimeMetric, Pi, Phi, GaugeH>;
+}  // namespace
+
+template <typename Solution>
+void verify_time_independent_einstein_solution(
```

Comment:
move this function to header file so it's simpler to use. We're not so concerned about compilation time in this case because you can compile all the cpp files for the tests in parallel anyway.